File: Lab_loop/t3.java
import java.util.Scanner;
public class t3{
    public static void main(String[] args){
        int n, counter = 1, result = 1;
        System.out.println("Please input n: ");
        Scanner scan = new Scanner(System.in);
        n = scan.nextInt();
        scan.close();
        if (n < 0){
            System.out.println("Please input a non-negative integer.");
            System.exit(-1);
        }
        while (counter <= n){
            result *= 2;
            counter++;
        }
        System.out.printf("The result of 2^%d is %d\n", n, result);
    }
}

File: Lab_loop/t3.cpp
#include <stdio.h>
int main(){
    int n, counter = 1, result = 1;
    puts("Please input n: ");
    scanf("%d", &n);
    if (n < 0){
        puts("Please input a non-negative integer.");
        return -1;
    }
    while(counter <= n){
        result *= 2;
        counter++;
    }
    printf("The result of 2^%d is %d\n", n, result);
    return 0;
}